package.json

```diff
@@ -64,8 +64,8 @@
     "@polymer/paper-toggle-button": "^3.0.1",
     "@polymer/paper-tooltip": "^3.0.1",
     "@polymer/polymer": "3.1.0",
-    "@vaadin/vaadin-combo-box": "^4.2.8",
-    "@vaadin/vaadin-date-picker": "^3.3.3",
```

Comment:
Don't forget the `vaadin` packageg that is pinned in `resolutions`, probably can drop it.